ArduCopter/land_detector.cpp

```diff
@@ -72,7 +72,9 @@ void Copter::update_land_detector()
                                     || ((!force_flying || landing) && motors->limit.throttle_lower && pos_control->get_vel_desired_cms().z < 0.0f);
 #else
         // check that the average throttle output is near minimum (less than 12.5% hover throttle)
-        bool motor_at_lower_limit = motors->limit.throttle_lower && attitude_control->is_throttle_mix_min();
+        bool motor_at_lower_limit = motors->limit.throttle_lower
+            // throttle is never at mix min in airmode
+            && (attitude_control->is_throttle_mix_min() || air_mode == AirMode::AIRMODE_ENABLED);
```

Comment:
needs to only make this change if in a manual throttle mode, otherwise increases risk of land-detect mid-air in modes like LAND

Reply:
Fixed